### ISSUE TITLE
Feature Request: Auto-hide Horizontal Scroll Bar

### ISSUE THREAD

**Issue by drkui** [babluboy/bookworm#284]
One feature that I would find as a nice addition is to have the option to auto-hide the horizontal scroll bar on the bottom when not in use in full screen mode (F11). Having it disappear after a second or so (or even stay hidden?) would be a more immersive experience.

**Comment by babluboy** [comment]
@drkui Thanks for raising this issue. I don't remember seeing a horizontal scroll bar - the flow should be to render the content vertically and have a vertical scrolbar only. I also donot remember adding any specific code to either show or prevent a horizontal scrollbar.

It will be great if you can attach the ebook to this issue for me to investigate this further.

**Comment by johnfactotum** [comment]
I think @drkui is probably referring to the progress bar, similar to #269?

**Comment by drkui** [comment]
Yes sorry, it's indeed the progress bar. I didn't realize there was an issue about this already, but yes it's similar. 

The two suggestions in #269 to either go into full screen or make it slimmer doesn't really address my specific feature request, which is to have the option to auto-hide the progress bar after a certain amount of time (since either way the bar is still there).

**Comment by babluboy** [comment]
Thanks @drkui and @johnfactotum for clarifying. We will continue to track this on #269 . Closing this Issue.

**Comment by babluboy** [comment]
@drkui I checked that in Full Screen mode the progress bar is not shown and you can navigate with arrow keys. Please check #63 which has already been implemented. You can enter ful screen either with F11 or from the right click context menu on the reading mode

**Comment by drkui** [comment]
@babluboy Thanks for following up, I rechecked and I'm still seeing the progress bar on the bottom in full screen mode.

![85VnqcO](https://user-images.githubusercontent.com/11448650/69824508-c84b6a00-11c0-11ea-9a3c-e5587f79e57e.png)

**Comment by babluboy** [comment]
@drkui That is strange. Can you check if your bookworm version is 1.1.2 or different? You can find the version from the About menu on the gear icon on top right

**Comment by drkui** [comment]
@babluboy Just checked and I do indeed have 1.1.2, installed from AUR.

**Comment by babluboy** [comment]
I have replicated the issue. This bug must have come in while impementing #194 
You can right click and select Enter Full Screen and then the progress bar is removed. However the context menu and F11 and Esc gets out of sync with each other causing this issue.

Will fix this soon.

**Comment by cyberjaguar** [comment]
same here, please let me remove that scrollbar, please! Thanks!
